We cannot run in max isolation because we need to set PYTHONPATH.

--- aiosignal-1.4.0/tox.ini.orig
+++ aiosignal-1.4.0/tox.ini
@@ -25,7 +25,6 @@
 commands =
   {envpython} \
     {[python-cli-options]byte-errors} \
-    {[python-cli-options]max-isolation} \
     {[python-cli-options]warnings-to-errors} \
     -m pytest \
       {tty:--color=yes} \
